src/components/Dropdown/PrioritySelector/__tests__/PrioritySelector.spec.tsx

```diff
@@ -0,0 +1,10 @@
+describe('PrioritySelector', () => {
+    it.todo('renders correctly with all props provided');
+    it.todo('should handle change correctly');
+    it.todo('should render correctly closed');
+    it.todo('should render correctly opened');
+    it.todo('should access only enabled items with value during keyboard navigation');
```

Comment:
I don't think this is relevant. There won't be anything without value, will there?